src/main/java/at/ac/tuwien/damap/validation/AccessValidator.java

```diff
@@ -56,4 +56,22 @@ public boolean canEditDmp(long dmpId, String personId) {
     public boolean canExportDmp(long dmpId, String personId) {
         return this.canViewDmp(dmpId, personId);
     }
+
+    public boolean canDeleteDmp(long dmpId, String personId) {
+        if (securityService.isAdmin()) {
+            return true;
+        }
+
+        List<Access> accessList = accessRepo.getAllDmpByUniversityId(personId);
+
+        for (Access access : accessList) {
+            if (access.getDmp().id.equals(dmpId) &&
+                    access.getRole().equals(EFunctionRole.OWNER)
+            ) {
+                return true;
+            }
+        }
+
+        return false;
```

Comment:
could use stream and optional for better readability
```suggestion
        Optional<Access> dmpAccess = accessList.stream().filter(access -> 
                access.getDmp().id.equals(dmpId) &&
                access.getRole().equals(EFunctionRole.OWNER)).findAny();
        
        return dmpAccess.isPresent();
```